app/assets/javascripts/discourse/app/components/flag-action-type.js

```diff
@@ -20,7 +20,7 @@ export default Component.extend({
   )
   formattedName(name, nameKey, isCustomFlag, username) {
     if (isCustomFlag) {
-      return name.replace("{{username}}", username);
```

Comment:
I wonder if we should temporarily support both, as this will break all non-english languages right away?

Reply:
Oh, good catch. I didn't think about that. I'll do that and remove it with the next update of the translations.